CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 ### Added
--
+- `RMSE` metric & `rmse` functional metric
```

Comment:
May you also add a ling to the current PR in the format "(\[#\<pr number\>\](link))". See the raw view of [CHANGELOG.md](https://raw.githubusercontent.com/tinkoff-ai/etna/master/CHANGELOG.md) file for the examples

Reply:
Done